### PR TITLE
Prepare confirmation for deployment

### PR DESCRIPTION
**Pull request by alvin-law**

### CODE REVIEW

File: app/client/views/confirmation/confirmation.html [hunk 1]
```diff
@@ -149,16 +162,6 @@
...
-
-            <div class="field"
-              ng-show="user.confirmation.reimbursementType == 'Airfare'">
-              <label> Are you a US citizen? </label>
-              <div class="ui checkbox">
-                <input type="checkbox" ng-model="user.confirmation.usCitizen">
-                <label> I am a US citizen. </label>
-              </div>
-            </div>
-
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmation.html]: Check this and where it got moved.

File: app/client/views/confirmation/confirmationCtrl.js [hunk 2]
```diff
@@ -81,6 +81,15 @@ angular.module('reg')
...
             },
+            wantsLaptop: {
+              identifier: 'wantsLaptop',
+              rules: [
+                {
+                  type: 'empty',
+                  prompt: 'Please give us a laptop preference!'
+                }
+              ]
+            },
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmationCtrl.js]: We don't need this anymore.

File: app/client/views/confirmation/confirmation.html [hunk 3]
```diff
@@ -215,122 +218,30 @@
...
-              Upload
-            </a>
-          </div>
+      		<div class="field">
+      		<label>Resume</label>
+      			<form id="resume" method="post" enctype="multipart/form-data">
+                    <input type="hidden"  name="key" value="{{user.email}}.pdf" />
+      				<input id="resume_file"name="file" type="file" accept=".pdf"/>
+      			</form>
+      			<h6 style="padding:none"><strong>You must submit your resume as a PDF, no exceptions!</strong></h6>
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmation.html]: Can we remove the "no exceptions" part LOL

Comment (neilmckibben) [reply app/client/views/confirmation/confirmation.html]: lmao remove dales legacy sure

**Pull request merged**